ui/src/pages/Install/index.tsx

```diff
@@ -294,7 +295,6 @@ const Index: FC = () => {
 
   useEffect(() => {
     configYmlCheck();
-    setupInstallLanguage(Storage.get(CURRENT_LANG_STORAGE_KEY));
   }, []);
```

Comment:
No need to adjust here.

You only need to adjust the FirstStep component and determine whether there is local storage for the language when the component is initialized, that is,

eg:
![image](https://github.com/user-attachments/assets/64048891-53fe-4527-b25d-ec46d70f7078)

Because the first value is taken by default here, it will not match the cached options.

Reply:
Using currentLang does not switch to that language by default.

This is just to show the correct language when the config file exists and step 6 or 7 is executed. So moved to configYmlCheck.

Reply:
This processing method does not meet the expectations of #1077, Maybe you can discuss with @LinkinStars 
whether this plan is feasible.

The solution I proposed above is tuned to the expectations of #1077

Reply:
okay, I got it.